Modules/BenchPress.Azure/Public/Confirm-EventHubConsumerGroup.ps1

```diff
@@ -0,0 +1,61 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-EventHubConsumerGroup {
+  <#
+    .SYNOPSIS
+      Confirms that an EventHub ConsumerGroup exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPEventHubConsumerGroup cmdlet gets an EventHub ConsumerGroup using the specified EventHub ConsumerGroup name,
+      EventHub Namespace name, Eventhub name and Resource Group name.
+
+    .PARAMETER Name
+      The name of the EventHub ConsumerGroup
+
+    .PARAMETER NamespaceName
+      The name of the EventHub Namespace
+
+    .PARAMETER EventHubName
+      The name of the EventHub
+
+    .PARAMETER ResourceGroupName
+      The name of the Resource Group
+
+    .EXAMPLE
+      Confirm-AzBPEventHubConsumerGroup -Name 'consumergrouptest' -NamespaceName 'bpeventhubnamespace' -EventHubName 'bpeventhub' -ResourceGroupName "rgbenchpresstest"
+
+    .INPUTS
+      System.String
+
+    .OUTPUTS
+      ConfirmResult
+  #>
+  [CmdletBinding()]
+  [OutputType([ConfirmResult])]
+  param (
+    [Parameter(Mandatory=$true)]
+    [string]$Name,
+
+    [Parameter(Mandatory=$true)]
+    [string]$NamespaceName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$EventHubName,
+
+    [Parameter(Mandatory=$true)]
+    [string]$ResourceGroupName
+  )
+  Begin {
+    $ConnectResults = Connect-Account
+  }
+  Process {
+    $Resource = Get-AzEventHubConsumerGroup -Name $Name -NamespaceName $NamespaceName -EventHubName $EventHubName -ResourceGroupName $ResourceGroupName
```

Comment:
This line is too long. Use splatting to overcome the length problem.

I'm not sure why megalinter isn't picking up the line length issues. We'll have to look into that.